cleanlab/classification.py

```diff
@@ -782,123 +791,160 @@ def find_label_issues(
         self.ps = value_counts(labels) / float(len(labels))
 
         self.clf_kwargs = clf_kwargs
-        self._process_label_issues_kwargs(self.find_label_issues_kwargs)
-        # self._process_label_issues_kwargs might set self.confident_joint. If so, we should use it.
-        if self.confident_joint is not None:
-            self.py, noise_matrix, inv_noise_matrix = estimate_latent(
-                confident_joint=self.confident_joint,
-                labels=labels,
-            )
-
-        # If needed, compute noise rates (probability of class-conditional mislabeling).
-        if noise_matrix is not None:
-            self.noise_matrix = noise_matrix
-            if inverse_noise_matrix is None:
+        if self.low_memory:
+            # If needed, compute P(label=k|x), denoted pred_probs (the predicted probabilities)
+            if pred_probs is None:
                 if self.verbose:
-                    print("Computing label noise estimates from provided noise matrix ...")
-                self.py, self.inverse_noise_matrix = compute_py_inv_noise_matrix(
-                    ps=self.ps,
-                    noise_matrix=self.noise_matrix,
+                    print(
+                        "Computing out of sample predicted probabilities via "
+                        f"{self.cv_n_folds}-fold cross validation. May take a while ..."
+                    )
+
+                pred_probs = estimate_cv_predicted_probabilities(
+                    X=X,
+                    labels=labels,
+                    clf=self.clf,
+                    cv_n_folds=self.cv_n_folds,
+                    seed=self.seed,
+                    clf_kwargs=self.clf_kwargs,
+                    validation_func=validation_func,
                 )
-        if inverse_noise_matrix is not None:
-            self.inverse_noise_matrix = inverse_noise_matrix
-            if noise_matrix is None:
-                if self.verbose:
-                    print("Computing label noise estimates from provided inverse noise matrix ...")
-                self.noise_matrix = compute_noise_matrix_from_inverse(
-                    ps=self.ps,
-                    inverse_noise_matrix=self.inverse_noise_matrix,
+
+            if self.verbose:
+                print("Using predicted probabilities to identify label issues ...")
+
+            arg_values = {
+                "thresholds": thresholds,
+                "noise_matrix": noise_matrix,
+                "inverst_noise_matrix": inverse_noise_matrix,
+                "find_label_issues_kwargs": self.find_label_issues_kwargs,
+            }
+            for arg_name, arg_val in arg_values.items():
```

Comment:
This check for `self.find_label_issues_kwargs` is invalid because by default `self.find_label_issues_kwargs = {}` and not `None`, so this warning will always trigger.

Reply:
That makes sense. So should I use a separate if-else statement just for `find_label_issues_kwargs`?

Reply:
Yeah you could, or you could also isolate `self.find_label_issues_kwargs` from that loop - whichever is easier, does not really matter too much